docusaurus/sidebar-connectors.js

```diff
@@ -332,8 +354,8 @@ function getDestinationConnectors(registry) {
 
   return [
     ...destinationsWithSupportLevel,
-    ...specialDestinationConnectors,
     ...enterpriseDestinationsWithSupportLevel,
+    destinationMsSql,
```

Comment:
since we created the item manually  in [L305](https://github.com/airbytehq/airbyte/pull/60233/files#diff-6f674668472168b20d96de38dce42c874367213f508826ad659dc8fc56fc993eR305)  and there's no exact match in the registry, we just add it to the array.